src/index.js

```diff
@@ -1,8 +1,21 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import firebase from 'firebase';
+
 import App from './App';
 import './index.css';
 
+
+const config = {
```

Comment:
also different from the example code on the firebase website, i'm using `const` here. what's nice about `const` (just not in this instance) is that it avoids the problem of a variable creeping out of scope, that is, this config var won't magically collide with other vars named `config` later on.